src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11Signature.java

```diff
@@ -119,6 +119,9 @@
     // key instance used, if init*() was called
     private P11Key p11Key;
 
+    // signature length expected or 0 for unknown
+    private int signatureLength;
```

Comment:
nit: use a shorter name, e.g. sigLen, so to fit in one line.

Reply:
Ok